activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -64,7 +64,23 @@ def self.build_mem_cache(*addresses) # :nodoc:
         addresses = addresses.flatten
         options = addresses.extract_options!
         addresses = ["localhost:11211"] if addresses.empty?
-        Dalli::Client.new(addresses, options)
+
+        pool_options = {}
+        pool_options[:size] = options[:pool_size] if options[:pool_size]
+        pool_options[:timeout] = options[:pool_timeout] if options[:pool_timeout]
+
+        if pool_options.present?
```

Comment:
`!empty?` we avoid to use `present?` inside the framework (unless we need to)

Reply:
I'm not sure if it is possible to add that check in Robocop but if it is, this review can be automated for future PRs 🤖

Reply:
Good idea. I'll take a look.